src/software/thunderscope/chicker/chicker.py

```diff
@@ -0,0 +1,186 @@
+            self.chip_button.setStyleSheet("background-color: Grey")
+            self.chip_button.setCheckable(False)
+            self.kick_button.setStyleSheet("background-color: Grey")
+            self.kick_button.setCheckable(False)
+        else:
+            self.charge_button.setStyleSheet("background-color: White")
+            self.charge_button.setCheckable(True)
+            self.chip_button.setStyleSheet("background-color: White")
+            self.chip_button.setCheckable(True)
+            self.kick_button.setStyleSheet("background-color: White")
+            self.kick_button.setCheckable(True)
+
+        # check all buttons
+        if self.charge_button.isChecked():
+            self.charge_button.toggle()
+            if self.charged:
+                print("Discharge clicked")
+                self.charged = False
+            else:
+                print("Charge clicked")
+                self.charged = True
+            print("Geneva:", self.geneva_value, "Power:", self.power_value)
+
+        if self.kick_button.isChecked():
+            self.kick_button.toggle()
+            self.charged = False
+            print("Kick clicked")
+            print("Geneva:", self.geneva_value, "Power:", self.power_value)
+
+        if self.chip_button.isChecked():
+            self.chip_button.toggle()
+            self.charged = False
+            print("Chip clicked")
+            print("Geneva:", self.geneva_value, "Power:", self.power_value)
+
+        # radio colors
+        if self.no_auto_button.isChecked():
+            self.radio_checkable[1] = True
+            self.radio_checkable[2] = True
+            if self.radio_checkable[0]:
+                print("No Auto clicked")
+                print("Geneva:", self.geneva_value, "Power:", self.power_value)
+            self.radio_checkable[0] = False
+            self.charge_button.setStyleSheet("background-color: White")
+            self.charge_button.setCheckable(True)
+            if self.charged:
+                self.chip_button.setStyleSheet("background-color: White")
+                self.chip_button.setCheckable(True)
+                self.kick_button.setStyleSheet("background-color: White")
+                self.kick_button.setCheckable(True)
+
+        elif self.auto_kick_button.isChecked():
+            self.radio_checkable[0] = True
+            self.radio_checkable[2] = True
+            if self.radio_checkable[1]:
+                print("Auto Kick clicked")
+                print("Geneva:", self.geneva_value, "Power:", self.power_value)
+            self.radio_checkable[1] = False
+            self.chip_button.setStyleSheet("background-color: Grey")
+            self.chip_button.setCheckable(False)
+            self.kick_button.setStyleSheet("background-color: Grey")
+            self.kick_button.setCheckable(False)
+            self.charge_button.setStyleSheet("background-color: Grey")
+            self.charge_button.setCheckable(False)
```

Comment:
I think we can pull this into a function like `def disable_buttons` and use it below as well